blockmanager.go

```diff
@@ -897,7 +891,7 @@ func (b *blockManager) getCheckpointedCFHeaders(checkpoints []*chainhash.Hash,
 
 				// We have another response to write, so delete
 				// it from the cache and write it.
-				queryResponses[checkPointIndex] = nil
+				delete(queryResponses, checkPointIndex)
```

Comment:
if we're actually removing the items, what do you think about having `queryResponses` check the `ok` value directly?

Reply:
Agreed. Done.